fastlane/Fastfile

```diff
@@ -117,18 +117,33 @@ platform :android do
     )
 
     UI.verbose("Deploying Custom Entitlements Computation version")
+    deploy_specific_package(
+      "customEntitlementComputationRelease",
+      "purchases-custom-entitlement-computation",
+      "purchases"
+    )
+
+    UI.verbose("Deploying Debug view version")
+    deploy_specific_package(
+      "defaultsDebug",
+      "purchases-debug-view",
```

Comment:
Users will be using the library like this: 
```
debugImplementation "com.revenuecat.purchase:purchases-debugview:6.10.0"
releaseImplementation "com.revenuecat.purchase:purchases-debugview-noop:6.10.0"
```

Reply:
Nice. We could document this [here](https://www.revenuecat.com/docs/debugging#debug-ui).

Reply:
Yup, once we deploy the first version, planning to add instructions on how to use this there 👍